app/components/rive-animation.ts

```diff
@@ -0,0 +1,81 @@
+import { action } from '@ember/object';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { Fit, Layout, Rive } from '@rive-app/canvas';
+import * as Sentry from '@sentry/ember';
+
+interface RiveAnimationSignature {
```

Comment:
```suggestion
interface Signature {
```

(Consistent with our other component files)